salt/modules/win_system.py

```diff
@@ -878,17 +912,29 @@ def set_system_date_time(years=None,
     if seconds is None:
         seconds = date_time[6]
 
-    # Create the time tuple to be passed to SetLocalTime, including day_of_week
-    time_tuple = (years, months, days, hours, minutes, seconds, 0)
-
     try:
-        win32api.SetLocalTime(time_tuple)
```

Comment:
@mollykreis Looks like you're recreating this below using ctypes.... Is there a problem witn `win32api.SetLocalTime`?